### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_Debugging_Resize: "Github Project"  link is getting displayed  inside the  hamburger button at 200% which is not present at 100%

### ISSUE THREAD

**Issue by Nikhitha-13** [microsoft/vscode-docs#2667]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment:**
code.visualstudio.com/docs/editor/debugging
Browser: Microsoft Edge 44.17763.1.0, Chrome 73.0.3683.103​

**NOTE:**
The issue is observed through out the application in the following URLs:

1. code.visualstudio.com/docs.
2. code.visualstudio.com/download.
3. code.visualstudio.com/docs/supporting/faq.
4. code.visualstudio.com/docs/setup/linux.
5. code.visualstudio.com/docs/getstarted/settings.
6. code.visualstudio.com/docs/python/python-tutorial.
7. code.visualstudio.com/updates/v1_30.
8. code.visualstudio.com/docs/editor/whyvscode
9. code.visualstudio.com/docs/getstarted/tips-and-tricks.
10. code.visualstudio.com/updates/v1_29.
11. code.visualstudio.com/docs/editor/integrated-terminal.
13. code.visualstudio.com/docs/editor/versioncontrol.

**Reprosteps:**
1. Hit the URL "code.visualstudio.com/docs/editor/debugging".
2. Resize the page to 200%.
3. Tab till the hamburger button and press enter.
4. Check whether we are getting the link 'GitHub project' or not.

**Actual Result:**
At 200% zooming, we are getting one extra link 'GitHub Project'  after FAQ link which is not present at 100%.

**Expected Result:**
GitHub Project link should not get displayed inside hamburger button at 200% since the link is actually not present at 100% after FAQ link.

**User Impact:**
Normal users will not be knowing about the 'GitHub Project' link unless they resize the page. so users will be missing the link.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Comment by Nikhitha-13** [comment]
[Bug_#2667_Resize issue.pptx](https://github.com/microsoft/vscode-docs/files/3165236/Bug_.2667_Resize.issue.pptx)

**Comment by Nikhitha-13** [comment]
![Bug_#2667_Github project resize issue](https://user-images.githubusercontent.com/50437925/57673021-299e5800-7638-11e9-8a4a-8986ba04001d.png)

**Comment by Nikhitha-13** [comment]
![Bug_#2667_github project issue](https://user-images.githubusercontent.com/50437925/57673136-8ef24900-7638-11e9-9e66-5cebcb73d3a4.png)

**Comment by Nikhitha-13** [comment]
![Bug_#2667_Resize issue](https://user-images.githubusercontent.com/50437925/57673253-e8f30e80-7638-11e9-82ab-2b347f61c629.png)

**Comment by gregvanl** [comment]
Extra "GitHub Project" link removed

**Comment by Nikhitha-13** [comment]
Checked the issue again. The link "GitHub Project" is now not visible at 200% under hamburger button

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by Roopesh-T** [comment]
**GitTag**: #closed; 
![image](https://github.com/microsoft/vscode-docs/assets/86601127/2fe15a50-bbce-4ab9-9df6-025102a9e072)

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
